src/machine/machine.c

```diff
@@ -323,11 +323,14 @@ int machine_load(Machine *m) {
                         int ifi;
 
                         r = extract_first_word(&p, &word, NULL, 0);
-                        if (r < 0)
-                                return log_error_errno(r, "Failed to parse NETIF: %s", netif);
-
                         if (r == 0)
                                 break;
+                        else if (r == 1)
+                                continue;
```

Comment:
No! If `r == 1`, `extract_first_word()` extracted a word, so we must of course *not* `continue` but execute the rest of the branch! That `continue` must live in an `r < 0` condition branch. This certainly hasn't been tested.

Please, test your patches under real conditions to make sure they do what they should.